e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -280,6 +301,45 @@ func (mts *MetricbeatTestSuite) installedUsingConfiguration(configuration string
 	return nil
 }
 
+func (mts *MetricbeatTestSuite) installOracleTools() error {
```

Comment:
If this is not working, wdyt about moving it to a different PR?

Reply:
At the end of the day, oracle is skipped, but I like the idea of not having dead code. Let me do it